scripts/launch/connect-to-cluster/default.nix

```diff
@@ -73,18 +73,21 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
   export LC_ALL=en_GB.UTF-8
   export LANG=en_GB.UTF-8
   if [ ! -d ${stateDir}/tls ]; then
-    mkdir ${stateDir}/tls/
-    ${pkgs.openssl}/bin/openssl req -x509 -newkey rsa:2048 -keyout ${stateDir}/tls/server.key -out ${stateDir}/tls/server.cert -days 3650 -nodes -subj "/CN=localhost"
+    mkdir -p ${stateDir}/tls/server && mkdir -p ${stateDir}/tls/client
+    ${iohkPkgs.cardano-sl-tools}/bin/cardano-x509-certificates   \
+      --server-out-dir ${stateDir}/tls/server                    \
+      --client-out-dir ${stateDir}/tls/client                    \
```

Comment:
Change this to `--clients-out-dir` and add `-e` to line 61 (`#!${pkgs.stdenv.shell} -e`).